src/allencell_ml_segmenter/_tests/prediction/test_model_input_widget.py

```diff
@@ -37,10 +37,6 @@ def test_top_radio_button_slot(
     # ASSERT - both input UI elements should be disabled, postprocessing method should be set to TOP_TEXT
```

Comment:
nit: if you're getting rid of assertions on postproc method, this reference to postprocessing method in the comment might be confusing

Reply:
Applies to comments throughout this file